apps/help-center/help-center-wp-admin.js

```diff
@@ -4,20 +4,29 @@ import { recordTracksEvent } from '@automattic/calypso-analytics';
 import HelpCenter from '@automattic/help-center';
 import { QueryClientProvider, QueryClient } from '@tanstack/react-query';
 import { useDispatch as useDataStoreDispatch, useSelect } from '@wordpress/data';
-import { useEffect, useCallback } from '@wordpress/element';
+import { useEffect, useCallback, useState } from '@wordpress/element';
 import { createRoot } from 'react-dom/client';
 const queryClient = new QueryClient();
 import './help-center.scss';
 
 function AdminHelpCenterContent() {
-	const { setShowHelpCenter, setShowSupportDoc } = useDataStoreDispatch( 'automattic/help-center' );
-	const { show, unreadCount } = useSelect( ( select ) => ( {
-		show: select( 'automattic/help-center' ).isHelpCenterShown(),
+	const { setShowHelpCenter, setShowSupportDoc, setNavigateToRoute } =
+		useDataStoreDispatch( 'automattic/help-center' );
+	const { isShown, unreadCount } = useSelect( ( select ) => ( {
+		isShown: select( 'automattic/help-center' ).isHelpCenterShown(),
 		unreadCount: select( 'automattic/help-center' ).getUnreadCount(),
 	} ) );
```

Comment:
We should pass `[]` as a second param to all `useSelect` calls.

Reply:
Done!